client/components/TestRun/index.jsx

```diff
@@ -367,7 +367,7 @@ const TestRun = () => {
                 'YY.MM.DD'
             )}`,
             testTitle: currentTest.title,
-            testRowNumber: currentTest.rowNumber,
+            testSequenceNumber: currentTest.seq,
```

Comment:
```suggestion
            testRowNumber: currentTest.rowNumber,
            testSequenceNumber: currentTest.seq,
```

Looks like `testRowNumber` should be re-added here to make sure the test number continues to be included when created from `<TestRun/>` as well, and to also preserve the `issuesResolver` search.

Reply:
Committed. Thank you!